components/audio_player/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+idf_component_register(SRCS "audio_player.cpp"
+                       INCLUDE_DIRS "include"
+                       REQUIRES bsp libhelix-mp3
```

Comment:
Could we remove the dependency on `bsp` ? If yes, the `audio_player` could be a nice reuseble component